content/docs/command-reference/exp/list.md

```diff
@@ -20,12 +20,12 @@ or commit they're based on. This is similar to `dvc exp show --no-pager`, but
 limited to experiment names and with very simple formatting. See also
 `dvc exp run`.
 
-If a working `git_remote` name (e.g. `origin`) or valid Git repo's URL is
-provided, lists experiments in that <abbr>repository</abbr> instead (if any,
-based on the `dvc remote default`).
+If a working `git_remote` name (e.g. `origin`) or valid Git URL is provided,
+lists experiments in that <abbr>repository</abbr> instead (if any, based on the
+`dvc remote default`).
```

Comment:
> if any, based on `dvc remote default`

Unrelated but is that bit correct? I think the experiments may be listed whether or not they've been pushed to the default remote or any DVC remote for that matter. Not sure

Reply:
Yes, whether the data was pushed to the dvc remote is irrelevant to `dvc exp list`. Cc @pmrowla for confirmation.